tests/pytest_tests/system_tests/test_core/test_wandb_init.py

```diff
@@ -115,6 +127,16 @@ def test_resume_never_failure(wandb_init):
     run_id = run.id
     run.finish()
 
+    # Wait for run metadata to finish uploading since resume logic relies on wandb-metadata.json being uploaded
+    api = Api()
+    api_run = api.run(f"{run.entity}/project/{run_id}")
+    metadata = None
+    tries = 0
+    while metadata is None and tries < 5:
+        metadata = api_run.metadata
+        time.sleep(1)
```

Comment:
same

Reply:
Do we have any guidance/tutorials/examples of how I should update this test to not use the old backend?

Reply:
sry, this referred to the two tests above

Reply:
Unless there is a mechanism for waiting on the server to handle file uploads I'm unaware of, the only alternative I can think of is mocking the server response for resume status for these tests.